src/application.c

```diff
@@ -119,6 +126,7 @@ static void action_update_color_state(struct swappy_state *state, double r,
   state->settings.g = g;
   state->settings.b = b;
   state->settings.a = a;
+  if (state->config->transparent) state->settings.a *= 0.5;
```

Comment:
This is just too much "magic number". Transparency, once enabled, should be configurable by the user.